x/gov/genesis.go

```diff
@@ -110,6 +110,13 @@ func InitGenesis(ctx sdk.Context, k Keeper, data GenesisState) {
 		k.setVote(ctx, vote.ProposalID, vote.Vote.Voter, vote.Vote)
 	}
 	for _, proposal := range data.Proposals {
+		status := proposal.GetStatus()
```

Comment:
Don't wanna be supppper nit-picky here but we can just switch on `proposal.GetStatus()` instead of having to create a variable. Also second what @fedekunze asked.